<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>625</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>821</width>
    <height>625</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="centralwidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>762</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="topMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>30</number>
     </property>
     <property name="spacing">
      <number>20</number>
     </property>
     <item row="0" column="0">
      <widget class="QFrame" name="statsFrame">
       <property name="styleSheet">
        <string notr="true">QFrame {
	background: white;
	border: 2px solid #bdc3c7;
	border-radius: 10px;
	padding: 20px;
}
QFrame:hover {
	border-color: #3498db;
	background: #f8f9fa;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="statsTitle">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 14pt;
font-weight: bold;
color: #2c3e50;
margin-bottom: 10px;</string>
          </property>
          <property name="text">
           <string>üìä –°—Ç–∞—Ç–∏—Å—Ç–∏–∫–∞</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="statsButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background: #3498db;
	color: white;
	border: none;
	padding: 8px 16px;
	border-radius: 5px;
	font-weight: bold;
	margin-top: 10px;
}
QPushButton:hover {
	background: #2980b9;
}</string>
          </property>
          <property name="text">
           <string>–û—Ç–∫—Ä—ã—Ç—å</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QFrame" name="usersFrame">
       <property name="styleSheet">
        <string notr="true">QFrame {
	background: white;
	border: 2px solid #bdc3c7;
	border-radius: 10px;
	padding: 20px;
}
QFrame:hover {
	border-color: #3498db;
	background: #f8f9fa;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="usersTitle">
          <property name="styleSheet">
           <string notr="true">font-size: 14pt;
font-weight: bold;
color: #2c3e50;
margin-bottom: 10px;</string>
          </property>
          <property name="text">
           <string>üë• –ü–æ–ª—å–∑–æ–≤–∞—Ç–µ–ª–∏</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="usersButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background: #3498db;
	color: white;
	border: none;
	padding: 8px 16px;
	border-radius: 5px;
	font-weight: bold;
	margin-top: 10px;
}
QPushButton:hover {
	background: #2980b9;
}</string>
          </property>
          <property name="text">
           <string>–£–ø—Ä–∞–≤–ª—è—Ç—å</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QFrame" name="sessionsFrame">
       <property name="styleSheet">
        <string notr="true">QFrame {
	background: white;
	border: 2px solid #bdc3c7;
	border-radius: 10px;
	padding: 20px;
}
QFrame:hover {
	border-color: #3498db;
	background: #f8f9fa;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="sessionsTitle">
          <property name="styleSheet">
           <string notr="true">font-size: 14pt;
font-weight: bold;
color: #2c3e50;
margin-bottom: 10px;</string>
          </property>
          <property name="text">
           <string>‚è±Ô∏è –ê–∫—Ç–∏–≤–Ω—ã–µ —Å–µ—Å—Å–∏–∏</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sessionsButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background: #3498db;
	color: white;
	border: none;
	padding: 8px 16px;
	border-radius: 5px;
	font-weight: bold;
	margin-top: 10px;
}
QPushButton:hover {
	background: #2980b9;
}</string>
          </property>
          <property name="text">
           <string>–ü—Ä–æ—Å–º–æ—Ç—Ä–µ—Ç—å</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QFrame" name="auditFrame">
       <property name="styleSheet">
        <string notr="true">QFrame {
	background: white;
	border: 2px solid #bdc3c7;
	border-radius: 10px;
	padding: 20px;
}
QFrame:hover {
	border-color: #3498db;
	background: #f8f9fa;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="auditTitle">
          <property name="styleSheet">
           <string notr="true">font-size: 14pt;
font-weight: bold;
color: #2c3e50;
margin-bottom: 10px;</string>
          </property>
          <property name="text">
           <string>üìã –ñ—É—Ä–Ω–∞–ª –∞—É–¥–∏—Ç–∞</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="auditButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background: #3498db;
	color: white;
	border: none;
	padding: 8px 16px;
	border-radius: 5px;
	font-weight: bold;
	margin-top: 10px;
}
QPushButton:hover {
	background: #2980b9;
}</string>
          </property>
          <property name="text">
           <string>–û—Ç–∫—Ä—ã—Ç—å</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>–í—ã—Ö–æ–¥</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>–û–±–Ω–æ–≤–∏—Ç—å</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
